client/components/domains/registrant-extra-info/fr-form.jsx

```diff
@@ -277,12 +276,12 @@ class RegistrantExtraInfoFrForm extends React.PureComponent {
 
 	renderOrganizationFields() {
 		const translate = this.props.translate;
-		const extra = this.props.contactDetails.extra;
+		const { extra } = this.props.contactDetails;
 		const {
 			registrantVatId,
 			sirenSiret,
-			trademarkNumber,
-		} = { ...emptyValues, ...extra };
+			trademarkNumber
+		} = defaults( extra, emptyValues );
```

Comment:
Ugh, yep. Thanks for catching.

I hate the way that works--not what one would intuitively expect. Destructuring should deep clone always. Maybe have secondary dedicated syntax for shallow cloning.